### PR TITLE
Add 1password7 extension

### PR DESCRIPTION
**Pull request by khasbilegt**
## Description

**This extension is based on @tdwesten 's original alpha codebase. Thank you for allowed me to port it to the public API.**

1Password 7 extension that allows you to view, edit and open & fill your passwords from Raycast. It consists of single command which lists all the passwords the user currently have. 

## Type of change

- New extension

## Screencast

output

## Checklist

- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

**Comment by tdwesten** [comment]
Thanks for upgrading my code!

File: extensions/1password7/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "1password7",
+  "title": "1Password 7 for Raycast",
```
Comment (PitNikola) [inline extensions/1password7/package.json:3]: No need to specify "for Raycast", it's implied. Let's just leave "1Password 7".

File: extensions/1password7/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  "description": "Search, open or edit your 1Password 7 passwords from Raycast",
+  "icon": "1password-icon.png",
+  "author": "khasbilegt",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "list",
+      "title": "My Passwords",
+      "subtitle": "Command",
```
Comment (PitNikola) [inline extensions/1password7/package.json:12]:
```suggestion
      "subtitle": "1Password",
```

File: extensions/1password7/src/list.tsx [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+
+// This is the official location, see for more information https://support.1password.com/integration-mac/
+const ONE_PASSWORD_7_CLI_FOLDER = `${os.homedir()}/Library/Containers/com.agilebits.onepassword7/Data/Library/Caches/Metadata/1Password`;
+const CACHE_DIR = environment.supportPath;
+const CACHE_FILE = `${CACHE_DIR}/cache.json`;
+const CLI_REQUIRED_MESSAGE = `
+# Spotlight and 3rd party app integrations is not enabled
+
+To use this extension please enable the "Spotlight and 3rd party app integrations" in the 1Password 7 app;
+1. Make sure you have te right 1Password app version, it should be 7.
```
Comment (PitNikola) [inline extensions/1password7/src/list.tsx:18]:
```suggestion
1. Make sure you have the right 1Password app version, it should be 7.
```

**Review by PitNikola — CHANGES_REQUESTED** [review]
Thanks for your contribution! Looks pretty good 👏, I only have few small minor suggestions we'll need to address before merging.

Also please use 512x512 icon, right now you have 256x256

**Comment by khasbilegt** [comment]
@PitNikola , I made the changes that you suggested 😎

**Pull request merged**